MarsQACompetitionTask/Pages/Certifications.cs

```diff
@@ -0,0 +1,180 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.ComponentModel;
+using System.Threading;
+using MarsQACompetitionTask.Utilities;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using System.Collections;
+
+
+namespace MarsQACompetitionTask.Pages 
+{
+    /*
+   public class Certifications : CommonDriver
+    {
+        [Test]
+        public void TestData()
+        {
+           driver.Navigate().GoToUrl(Utilities.ReadJsonData.GetData("Education[0].country"));
+           Console.WriteLine(Utilities.ReadJsonData.GetData("Education[0].country"));
+        }
+       
+    }*/
+    public class Certification
+    {
+        private readonly IWebDriver driver;
+        public Certification(IWebDriver driver)
+        {
+            this.driver = driver;
+        }
+        public void CertificationTab()
+        {
+            IWebElement CertificationTab1 = driver.FindElement(By.XPath("/html/body/div[1]/div/section[2]/div/div/div/div[3]/form/div[1]/a[4]"));
+            CertificationTab1.Click();
+            //Thread.Sleep(10000);
+        }
+        public void CreateCertification()
+        {
+            //Thread.Sleep(2000);
+
+            IList certdetails;
+            certdetails = Utilities.ReadJsonData.GetDataObject2("Certification[*]");
+            int j = 0;
+            j = certdetails.Count;
+
+            Console.WriteLine("cert details '''''", j);
+
+
+            for (int i = 0; i < j; i++)
+            { 
+
+                IWebElement AddNewBtn = driver.FindElement(By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[5]/div[1]/div[2]/div/table/thead/tr/th[4]/div"));
+            AddNewBtn.Click();
+            IWebElement CName = driver.FindElement(By.XPath("//input[@placeholder='Certificate or Award']"));
```

Comment:
Please implement the POM pattern 
https://docs.specflow.org/projects/specflow/en/latest/ui-automation/Selenium-with-Page-Object-Pattern.html 
The above is a specflow example but you should can see the page part and see exactly how it uses the pom pattern! The variable are initialised out side methods l

Reply:
This comment applys to all pages therefore fix all!